Tools/Scripts/run-mvt-tests

```diff
@@ -289,18 +326,45 @@ def run_mvt_test_suite(args):
     num_suites = len(suites)
     for suite in suites:
         json_results = mvtwebdriver_runner.run_suite(suite)
-        tests_run, unexpected_failures, expected_failures = mvtresultsexpectations_parser.analyze_results(json_results)
-        test_results[suite] = [tests_run, unexpected_failures, expected_failures]
+        tests_run_list, unexpected_failures_list, expected_failures_list = mvtresultsexpectations_parser.analyze_results(json_results)
+        unexpected_failures = len(unexpected_failures_list)
+        tests_run = len(tests_run_list)
+        # Maybe retry unexpected failures
+        if args.retry_unexpected_failures > 0 and unexpected_failures > 0:
+            unexpected_failures_str = "', '".join(unexpected_failures_list)
+            _log.info(f"Retrying {unexpected_failures} unexpected failures: '{unexpected_failures_str}'")
+            # iterate over a copy since we are modifying unexpected_failures_list inside the loop
+            for test_name in list(unexpected_failures_list):
+                repeat_test_results = mvtwebdriver_runner.run_test_from_suite(test_name, suite, args.retry_unexpected_failures)
+                _log.debug(f"Got extra results for {suite}/{test_name}: {repeat_test_results}")
+                for extra_result in repeat_test_results:
+                    if extra_result == 'passed':
+                        _log.info(f'[FLAKY][PASS] {test_name}.')
+                        unexpected_failures_list.remove(test_name)
+                        break
+                    elif not mvtresultsexpectations_parser._is_unexpected_failure(suite, test_name, extra_result):
+                        _log.info(f'[FLAKY][FAIL][EXPECTED] {test_name}')
+                        unexpected_failures_list.remove(test_name)
+                        expected_failures_list.append(test_name)
+                        break
+        # Calculate results of the suite run
+        unexpected_failures = len(unexpected_failures_list)
+        expected_failures = len(expected_failures_list)
+        test_results[suite] = [tests_run_list, unexpected_failures_list, expected_failures_list]
         exit_code = unexpected_failures
         tests_pass = tests_run - unexpected_failures - expected_failures
         suite_summary_header_str = f"[Suite {suite}] Ran {tests_run} tests of which:"
         _log.info('-' * len(suite_summary_header_str))
         _log.info(suite_summary_header_str)
         _log.info(f"    - {tests_pass} tests passed.")
         if expected_failures > 0:
-            _log.info(f"    - {expected_failures} tests were expected failures.")
+            _log.info(f"    - {expected_failures} tests were expected failures:")
```

Comment:
I think you could use https://docs.python.org/3/library/string.html#formatspec to add the padding / width instead